schemas/rdf/examples/AssetAdministrationShell_Example.ttl

```diff
@@ -9,65 +9,56 @@
-      <https://admin-shell.io/aas/3/0/RC01/Referable/description> "A very short description of the AAS instance"@en ;
+      <https://admin-shell.io/aas/3/0/RC02/Referable/description> "A very short description of the AAS instance"@en ;
       rdfs:comment "A very short description of the AAS instance."^^xsd:string ;
-      <https://admin-shell.io/aas/3/0/RC01/AssetAdministrationShell/assetInformation> [
-          a <https://admin-shell.io/aas/3/0/RC01/AssetInformation> ;
-          <https://admin-shell.io/aas/3/0/RC01/AssetInformation/assetKind> <https://admin-shell.io/aas/3/0/RC01/AssetKind/Instance> ;
-          <https://admin-shell.io/aas/3/0/RC01/AssetInformation/globalAssetId> [
-            a  <https://admin-shell.io/aas/3/0/RC01/Reference> ;
-              <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-                a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-                <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Asset> ;
-                <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://customer.com/assets/KHBVZJSQKIY"^^xsd:string ;
-                <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri> ;
+      <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/assetInformation> [
+          a <https://admin-shell.io/aas/3/0/RC02/AssetInformation> ;
+          <https://admin-shell.io/aas/3/0/RC02/AssetInformation/assetKind> <https://admin-shell.io/aas/3/0/RC02/AssetKind/Instance> ;
+          <https://admin-shell.io/aas/3/0/RC02/AssetInformation/globalAssetId> [
+            a  <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
+              <https://admin-shell.io/aas/3/0/RC02/ModelReference/key> [
+                a <https://admin-shell.io/aas/3/0/RC02/Key> ;
+                <https://admin-shell.io/aas/3/0/RC02/Key/type> <https://admin-shell.io/aas/3/0/RC02/KeyElements/Asset> ;
+                <https://admin-shell.io/aas/3/0/RC02/Key/value> "http://customer.com/assets/KHBVZJSQKIY"^^xsd:string ;
               ]
           ]
       ] ;
 
-      <https://admin-shell.io/aas/3/0/RC01/AssetAdministrationShell/submodel> [
-          a <https://admin-shell.io/aas/3/0/RC01/Reference> ;
-          <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-            a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Submodel>  ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://i40.customer.com/type/F13E8576F6488342"^^xsd:string ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>;
+      <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/submodel> [
+          a <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
+          <https://admin-shell.io/aas/3/0/RC02/ModelReference/key> [
```

Comment:
really key or keys?

Reply:
On page 87 of Spec, it is key, not keys.